Updated docs for roles and permissions (#219)
validmind/documentation: site/guide/_manage-permissions.qmd

> 1. Under User Permissions, select **Permissions**

Reviewer: ```suggestion
1. Under User Permissions, select **Permissions**.
```

> 1. Under your desired action, add or delete roles. 

Reviewer 2: How do I this fam, I've never seen the interface before, where do I click, give me some more context, please give me a step-by-step, or maybe even just break it down into bullet points:

e.g.
```
Manage roles that have authorization to perform actions:

- To assign a role to a permission, blah blah blah
- To remove a role from a permission, blah blah blah```

Reviewer: To add to what Beck said, the steps should cover that you click **+** to add a role to the permission or remove a role by clicking on the **x** on the label: 

<img width="991" alt="image" src="https://github.com/validmind/documentation/assets/15148011/b42c1883-8afa-498e-b86e-7add2252b60b">

> 1. Under your desired action, add or delete roles. 

Reviewer: ```suggestion
1. Add or delete roles that hold a specific permission:
```

I would then include the bulleted list you have in your comment/question and add an example below that: 

```
   - To assign a role to a permission, blah blah blah
   - To remove a role from a permission, blah blah blah

   For example, to grant a developer the permission to update an existing template ... yada yada yada 
```

Also, wording like `Under your desired action` doesn't really add anything, I'd just skip in favour of direct wording — what do I desire if I am a new user?
